back/engines/commercial/custom_maps/app/controllers/custom_maps/web_api/v1/project_map_configs_controller.rb

```diff
@@ -42,7 +42,12 @@ def show
           @project = Project.includes(map_config: %i[layers]).find(params[:project_id])
           authorize @project
           @map_config = @project.map_config
-          render json: serialized_map_config, status: :ok
+
+          if @map_config.nil?
+            head :no_content
+          else
+            render json: serialized_map_config, status: :ok
+          end
```

Comment:
When there is no map config, before we were returning a response with `{ data: null }` in the body. This was crashing our cashing system and leading to problems locally when you have no global nor project-level map config (you will see the map constantly refreshing in a strange way when clicking the toggle).